packages/helix-shared-indexer/src/index-resource.js

```diff
@@ -77,6 +82,7 @@ const helpers = {
     return [text.split(/\s+/g).slice(start, end).join(' ')];
   },
   replace: (s, searchValue, replaceValue) => [s.replace(searchValue, replaceValue)],
+  replaceAll: (s, searchValue, replaceValue) => [s.replaceAll(searchValue, replaceValue)],
```

Comment:
introduce `replaceAll` which does replace *all* occurrences, as stated wrongly in the documentation about `replace`